client/src/main/java/org/eclipse/hono/client/ApplicationClientFactory.java

```diff
@@ -145,19 +145,18 @@ static ApplicationClientFactory create(final HonoConnection connection) {
      * adapters need to convey an encoding of the reply-to address to the device when delivering
      * the command. Consequently, the number of bytes transferred to the device depends on the
      * length of the reply-to address being used. In situations where this is a major concern it
-     * might be advisable to use {@link #getOrCreateCommandClient(String, String, String)} for
+     * might be advisable to use {@link #getOrCreateCommandClient(String, String)} for
```

Comment:
IMHO the whole JavaDoc needs to be revised to reflect that the client is no longer scoped to a specific device anymore.

Reply:
Indeed, I'll change that.